net.sonic_pi.SonicPi.json

```diff
@@ -364,9 +375,10 @@
             ],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://github.com/sonic-pi-net/sonic-pi/archive/refs/tags/v4.4.0.tar.gz",
-                    "sha256": "4d00409f9bc8027579b6120ee537a4c54e0a2fae7b5501dcf3c810ff05c02bab"
+                    "//": "TODO: Point to 4.5.0 tarball when it's released",
+                    "type": "git",
+                    "url": "https://github.com/sonic-pi-net/sonic-pi.git",
+                    "branch": "dev"
```

Comment:
point to a commit instead. bare branches are not allowed. even for testing

Reply:
I just found out :roll_eyes: Makes sense but didn't think about it, since it worked locally :+1:

Reply:
```
        "module-sonicpi-source-git-no-commit-or-tag",
        "module-sonicpi-source-git-branch"
```